README.rst

```diff
@@ -13,3 +13,131 @@ __ https://discuss.openedx.org/c/working-groups/build-test-release/30
 More details about the working group are at the `BTR page in the wiki`__.
 
 __ https://openedx.atlassian.net/wiki/spaces/COMM/pages/1022099494/Build+-+Test+-+Release+Working+Group
+
+================================
+Role definitions and assignments
+================================
+
+*Current release: Lilac*
+
+The following roles, with corresponding expectations and assignments, are
+defined for the current release cycle.
+
+Group Chair
+===========
+
+*Current assignee: @arbrandes*
+
+The group chair is responsible for coordinating efforts that culminate in
+getting a new release out and maintaining the current one.  In particular, the
+Group Chair will:
+
+* Manage the current release cycle in general, including making sure that
+  proper deadlines are established and adhered to
+* Facilitate the assignment of roles and tasks
+* Coordinate the group's communication efforts, including running periodic
+  meetings such as the Biweekly Meetup, and summarizing recent happenings in
+  the Biweekly Update
+* Try and help solve problems as they come up, bringing them to the attention
+  of the right group members
+
+Release Manager
+===============
+
+*Current assignees: @nedbat and @arbrandes*
+
+The Release Manager is responsible for the actual cutting of a release, which,
+in essence, includes following the steps in the `Process to Create an Open edX Release
+<https://openedx.atlassian.net/wiki/spaces/COMM/pages/19662426/Process+to+Create+an+Open+edX+Release>`_.
+
+In addition, the Release Manager will:
+
+* Create public test or release candidate branches as required by the release process
+* Merge reviewed PRs into the release master branch (for instance,
+  ``koa.master``)
+* Notify the Release Testing Coordinator whenever additional testing is needed
+  in response to the above
+* Tag releases, major and minor, at the appropriate time
+
+Release Testing Coordinator
+===========================
+
+*Current assignee: @jfavellar90*
+
+The Release Testing Coordinator is responsible for making sure release
+candidates are tested _before_ a tag is made.  The coordinator does not
+necessarily have to run tests themselves, but they will:
+
+* Coordinate testing in general, in particular in the days leading to a timed
+  release, including the bimonthly minor ones.
+* Keep tabs on whatever CI is available, making sure it is running correctly
+* Advocate for better testing tools and environments, when necessary
+* Coordinate the reaction to test failures, such as bringing them to the
+  attention of the rest of the group by opening tickets or commenting in the forum
+
+Release Documentation Expert
+============================
+
+*Current assignee: N/A*
+
+The Release Documentation Expert is tasked with building the Release Notes
+document prior to a major release.  It involves:
+
+* Tracking down all relevant information,
```

Comment:
Can we narrow this scope and make it more specific? Perhaps "Gather information from the community about new features and breaking changes since the last release"

Reply:
I guess this depends what you mean by "from the community"?  The key source of information for the last few releases has been summaries from the edX Product organization.  With the improved pull request template and soon Conventional Commits, we're hoping we'll be able to mine those information sources.

Reply:
Great. I was hoping you'd say that. Do you think there's a reasonable possibility that we (I) could build a list of new features and breaking changes by reviewing commit messages and PRs? Perhaps that's too optimistic and we need two bullets here:

* Reviewing commits and pull requests across the open edX repositories. 
* Collecting a summary of new features from the edX Product organization.

Reply:
I guess my feeling is:
* the Juniper/Koa process was not great, and
* the pull request template and conventional commits are the right direction but haven't proven themselves,
so I'm hesitant to over-describe the current state.  We have more iterating to do before we know we have a process that works well.

Reply:
@pdpinch, this is by necessity going to be a best-effort kind of thing; I don't think anybody expects the release notes to be perfect. I reworded it to that effect.

Plus, it's going to be reviewed by all of us (hopefully) prior to release, so as to minimize the chance we miss something important.